components/browse/BrowseContentCard.vue

```diff
@@ -34,24 +35,33 @@
       ...mapGetters({
         apiConfig: 'apis/config'
       }),
+      cardFields() {
+        return this.cardType === 'automatedRecordCard' ? this.fields.encoding : this.fields;
+      },
       title() {
-        return this.fields.name;
+        if (this.cardType === 'automatedRecordCard') {
+          return this.cardFields.dcTitleLangAware;
+        }
```

Comment:
This won't be backwards compatible, do a check as above where it uses dcTitleLangAware if encoding is set.